_Streams/121-map-reduce-in-javascript.md

```diff
@@ -24,11 +24,11 @@ videos:
   - title: "CircleCI: Unit Testing and CI"
     url: "https://www.youtube.com/watch?v=CB7vnoXI0pE"
   - title: "ES6 Arrow Function"
-    url: "https://www.youtube.com/watch?v=mrYMzpbFz18"
+    url: "http://thecodingtrain.com/Tutorials/16-javascript-es6/16.3-arrow-function.html"
```

Comment:
You almost got it. Please just remove the `http://thecodingtrain.com` and `.html` part. The URL would then be: `/Tutorials/16-javascript-es6/16.3-arrow-function`